### PR TITLE
2020 workshops read me initial changes

### PR DESCRIPTION
**Pull request by rprakitpong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,22 +20,24 @@ Pronterface; EEPROM; Gcode; Marlin 1.1.6; Prusa own firmware; Configuration.h an
...
+|1|Friend|
+|2|Andrew|
+|3|Monica|
+|4|Monica|
+|5|Friend|
+|6|Friend|
+|7|Andrew|
+|8|Friend|
+|9|Friend|
+|Project|All|
```
Comment (rprakitpong) [inline README.md:40]: hey is this correct? can you change xyz?

File: README.md [hunk 2]
```diff
@@ -20,22 +20,24 @@ Pronterface; EEPROM; Gcode; Marlin 1.1.6; Prusa own firmware; Configuration.h an
...
 Navigating view; Basic model manipulation; Modifier intro (boolean); Export technique
- 8. Sustainability:
+ 8. Basic software (C++):
+TBA
+ 9. Sustainability:
 Ways to reduce, reuse, recycle when 3D printing; Ways to be sustainble in life
 
 ### Schedule
-|Week|Group A (Wed 6-7:00pm)|Group B (Tues 5:30-6:30pm)|
-|---|---|---|
```
Comment (AndrewSiew) [inline README.md:28]: This is Andrew

**Review by rprakitpong — COMMENTED** [review]
wow no good. go change stuff.

File: README.md [hunk 3]
```diff
@@ -60,12 +62,17 @@ I expect pre-workshop submissions 24 hours before workshop, and workshop submiss
...
--ws2 explain parametric model example
--ws1 cura download guide, fix to v3.6
--simplify readme
+- ws1 change filament, flush abs
+- rename ws from int to char
+- ws2 recommend non sldr options
+- ws2 explain parametric model example
+- ws1 cura download guide, fix to v3.6
+- simplify readme
+
```
Comment (MonicaNam) [inline README.md:71]: looks good :)

File: README.md [hunk 4]
```diff
@@ -20,22 +20,24 @@ Pronterface; EEPROM; Gcode; Marlin 1.1.6; Prusa own firmware; Configuration.h an
 Import and export files, navigating; Repairing holes in models; Smoothing; Basic Retopologizing; Cutting models; Tree support
  7. Basic Mesh Modeling (Blender):  
 Navigating view; Basic model manipulation; Modifier intro (boolean); Export technique
- 8. Sustainability:
+ 8. Basic software (C++):
+TBA
+ 9. Sustainability:
```
Comment (rprakitpong) [inline README.md:25]: sad

**Review by rprakitpong — COMMENTED** [review]
overall good.

File: README.md [hunk 5]
```diff
@@ -60,12 +62,17 @@ I expect pre-workshop submissions 24 hours before workshop, and workshop submiss
...
--ws2 recommend non sldr options
--ws2 explain parametric model example
--ws1 cura download guide, fix to v3.6
--simplify readme
+- ws1 change filament, flush abs
+- rename ws from int to char
+- ws2 recommend non sldr options
+- ws2 explain parametric model example
+- ws1 cura download guide, fix to v3.6
+- simplify readme
```
Comment (MonicaNam) [inline README.md:70]: nice

**Review by MonicaNam — APPROVED** [review]
good change

**Review by AndrewSiew — APPROVED** [review]
Good

**Pull request merged**